docs/coding.rst

```diff
@@ -170,30 +170,29 @@ in our `ESLint config <https://github.com/mozilla/bedrock/blob/master/.eslintrc.
 Writing URL Patterns
 --------------------
 
-URL patterns should be as strict as possible. It should begin with a
-`^` and end with `/$` to make sure it only matches what you specifiy.
-It also forces a trailing slash. You should also give the URL a name
-so that other pages can reference it instead of hardcoding the URL.
-Example:
+URL patterns should be as strict as possible. It also forces a trailing slash.
+You should also give the URL a name so that other pages can reference it instead
+of hardcoding the URL. Example:
```

Comment:
This reads a bit awkward to me. Is "It" referring to URL patterns? Also I don't think this matches the code any longer, the trailing slash is more explicitly defined in each URL now but optional still, it looks like.

Reply:
Should no longer be optional. I removed the optional addition of the URL as well.

Reply:
Updated wording.